src/app/views/provisioning/order.html

```diff
@@ -31,26 +31,26 @@ <h1 translate>{{ vm.subscriptionPlanText }}</h1>
         </div>
 
         <div class="row td-row" ng-repeat="pricingPlan in vm.filteredPricingPlans" ng-click="vm.selectPlan(pricingPlan)">
-          <div class="cell radio-selector col-xs-2">
+          <div class="cell radio-selector col-xs-1">
             <input type="radio" name="optradio" ng-model="vm.subscription.product_pricing.id" ng-value="pricingPlan.id" required>
           </div>
 
-          <div class="col-xs-10">
-            <div class="cell col-sm-2" uib-tooltip="{{ pricingPlan.name }}">
+          <div class="col-xs-11">
+            <div class="cell col-sm-3" uib-tooltip="{{ pricingPlan.name }}">
```

Comment:
Same here, I think we need a `row` around the nested columns

Also I only count 10 columns being used?

Reply:
This hasn't been addressed?

Reply:
Ok 12 columns now